.github/actions/regression-tests/action.yml

```diff
@@ -47,13 +52,12 @@ runs:
         # used by PyTests
         export DRAGONFLY_PATH="${GITHUB_WORKSPACE}/${{inputs.build-folder-name}}/${{inputs.dfly-executable}}"
 
-        pytest --json-report --json-report-file=rep1_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=true
-        pytest --json-report --json-report-file=rep2_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=false
-
+        pytest -m "${{inputs.filter}}" --json-report --json-report-file=rep1_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=true
+        pytest -m "${{inputs.filter}}" --json-report --json-report-file=rep2_report.json -sv dragonfly/replication_test.py --df alsologtostderr --df enable_multi_shard_sync=false
 
 
     - name: Send notification on failure
-      if: failure()
+      if: failure() && ${{ inputs.run-only-on-ubuntu-latest == 'true'}}
```

Comment:
why only on run-only-on-ubuntu-latest ?

Reply:
I think there might have a been a problem with the other image, maybe pip/pip3 or something silly like that